test/e2e/edu-5490-helpers.js

```diff
@@ -0,0 +1,150 @@
+import _ from 'lodash/fp';
+const Timeouts = require('./timeouts.js');
+
+
+// Same as 1990e
+// TODO: relocate to keep it DRY
+function completeEmploymentHistory(client, data, onlyRequiredFields) {
+  if (!onlyRequiredFields) {
+    const nonMilitaryJobs = data.nonMilitaryJobs[0];
+    client
+      .click('input[name="root_view:hasNonMilitaryJobsYes"]')
+      .clearValue('input[name="root_nonMilitaryJobs_0_name"]')
+      .setValue('input[name="root_nonMilitaryJobs_0_name"]', nonMilitaryJobs.name)
+      .clearValue('input[name="root_nonMilitaryJobs_0_months"]')
+      .setValue('input[name="root_nonMilitaryJobs_0_months"]', nonMilitaryJobs.months)
+      .clearValue('input[name="root_nonMilitaryJobs_0_licenseOrRating"]')
+      .setValue('input[name="root_nonMilitaryJobs_0_licenseOrRating"]', nonMilitaryJobs.licenseOrRating);
+  }
+}
+
+// This section is required, unlike the regular EduHelpers
+// Also actually uses the data
+function completeBenefitSelection(client, data) {
```

Comment:
I'll double-check, but I think I tried that first and it would have taken a lot of modification or something...